src/circuit.mjs

```diff
@@ -310,6 +310,19 @@ export class HeadlessCircuit {
         ret.subcircuits = subcircuits;
         return ret;
     }
+    getSvgExportUrl(circuitSvgDOMElement) {
```

Comment:
* Why is it added to the headless circuit class?
* Does this really work headless (without a JointJS paper)?
* What is the meaning of the `circuitSvgDOMElement` argument? The code suggests that it's related to the JointJS paper.

Reply:
- I moved this method to the `Circuit` class.
- Now the method takes `joint.dia.paper` as a parameter.